README.md

```diff
@@ -166,8 +168,11 @@ Check out [Speasy documentation](https://speasy.readthedocs.io/en/stable/) and [
 
 ### Caveats
 
--   Speasy is not a plotting package, while it provides basic plot
-    features, it is not meant to produce publication ready figures.
+-   Speasy is not a plotting package.
+    basic plotting capabilities are here for illustration purposes and making quick-and-dirty plots.
+    It is not (and will not be) meant to produce publication ready figures.
+    If you want to browse efficiently and graphically data, we recommand you check out [SciQLOP](https://github.com/SciQLop/SciQLop)
+    (which uses Speasy internally).
```

Comment:
Not sure this is the place to talk about SciQLop, this warning is about full featured production ready plots, the message is "use something like matplotlib directly". We could probably advertise about SciQLop in some Speasy usage section.